mlir/lib/Dialect/Rock/Transforms/AlignTiling.cpp

```diff
@@ -1145,21 +1148,399 @@ ReduceRewritePattern::matchAndRewrite(rock::ReduceOp reduceOp,
   }
   rewriter.moveAfterIfNeeded(threadwiseWriteOp, reduceOp);
 
+  // Obtain blockwise subtile.
+
+  // This has < block dimensions ... > x tid x iter to Gemm Dimensions.
+  ArrayAttr extraViews = threadwiseWriteOp.getExtraViews();
+  ArrayAttr destTrs;
+  Value dest;
+  std::tie(dest, destTrs, std::ignore) =
+      untransform(rewriter, threadwiseWriteOp.getDest());
+  ArrayAttr toBeReducedViews = prependUpperViews(rewriter, extraViews, destTrs);
+  TransformMapAttr firstCoordTransform =
```

Comment:
extraViews can never be empty by definition because those correspond to g_block x m_block x n_block x tid x iter -> G X M X N. (though destTrs could be empty, thus concat of the two will also never be empty)

Reply:
In the future, we might get rid of `extraViews`, for example

Reply:
that would mean they would go to destTrs.. right? 
coz somewhere there needs to hold the maps that starts with g_block x m_block x n_block x tid x iter -> G X M X N
So the concat would still ne not empty.. 

(I can put an assert)